index.src.html

```diff
@@ -781,17 +810,29 @@ <h3 id="try-delivery" algorithm>
 
       ISSUE: The "`report`" type does not exist in Fetch. Talk to Anne.
 
-  4.  <a>Queue a task</a> to <a>fetch</a> |request|.
+  4.  User agent MAY choose to alter |request| in order to send |report| to a
+      different endpoint.
+
+  Note: The intended use case for allowing reports to be sent to a different
+  endpoint is so that reports containing sensitive information (either for
+  security or privacy reasons) can be aggregated and anonymized at a third party
```

Comment:
I'm worried about this part.

Who gets to decide which third-party server to use?  This seems like it violates one of the security principles we've been designing around — that it's *only* the server administrator who gets to see reports about their domain.  (Or more accurately, that gets to decide who gets to see those reports.)  Doesn't this allow the user agent to upload all reports to some other server, against the wishes of the server admin?

If the reports contain potentially sensitive information, the spec controlling that report type should define how to aggregate or anonymize (in a way that all user agents can implement consistently) *before* the reports are handed off to Reporting for delivery.

Another way to put this: when I was reading the your definition of "aggregated type", I was thinking you were going to define a generic aggregation mechanism here.  (Group reports by ignoring any fields that are in the report type but not the aggregated type, recording the number of reports in each group.)  Then you wouldn't need the third-party service for aggregation; the Reporting implementation would just do the right thing before sending the (now aggregated) reports to the original collectors defined by the server admin.